src/primitive-field/primitive-field.component.ts

```diff
@@ -106,11 +102,15 @@ export class PrimitiveFieldComponent extends AbstractFieldComponent implements O
     this.commitValueChange();
   }
 
-  getTabIndex(): number {
-    return this.tabIndexService.getElemTabIndex(this.path.join('.'));
+  get linkBuilder(): LinkBuilderFunction {
```

Comment:
You're getting travis error because you haven't rebased, also this part is unrelated and actually is part of another PR.

@zzacharo